contents/newsletter/communication-mistakes.md

```diff
@@ -0,0 +1,202 @@
+---
+title: What engineers get wrong about communication
+date: 2025-03-23
+author:
+ - ian-vanagas
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/image_1_2594402957.png
+featuredImageType: full
+tags:
+  - Engineering
+crosspost:
+  - Product engineers
+  - Blog
+---
+
+Engineers spend most of their time doing two things: coding and communicating. The first has endless amounts written about it; the second much less so.
+
+To address this injustice, we're sharing:
+
+1. The biggest mistakes we see engineers make when communicating.
+
+2. What we do as a [remote](https://newsletter.posthog.com/p/habits-of-effective-remote-teams), [async](https://newsletter.posthog.com/p/how-we-work-asynchronously), [engineering-led](https://newsletter.posthog.com/p/the-magic-of-small-engineering-teams) company to eliminate them.
+
+## 1. Forgetting about your users
+
+It’s easy to get caught up in the technical details of a project, such as overcoming constraints, optimizing performance, and following best practices.  
+
+But companies don’t succeed based on their ability to solve [technical problems](https://newsletter.posthog.com/p/how-we-choose-technologies). They win when they solve their user’s [most painful and valuable problems](https://newsletter.posthog.com/p/how-to-uncover-your-users-real-problems).  
```

Comment:
I've probably missed this in the, but we should link to the website versions of newsletters on our website.

Reply:
got it